tools/README.md

```diff
@@ -0,0 +1,21 @@
+# Archive Node Developer Tools
+
+<hr>
+
+Some tools to make working with Mina Archive Nodes and their SQL dumps provided by O(1) a little easier
+
+<hr>
+
+## Download Archive Dump
+
+downloads the latest archive dump for your selected network
+
+* build: `nix-build`
+* run: `./result/bin/download-archive-dump --help`
+
+## Run Archive Node from Archive Dump
+
+create a new postgres instance and initialize it using a Mina archive dump
```

Comment:
Prefer proper English sentences, but I won't hold my approval on this.

Reply:
You really should say where the new PostgreSQL (note spelling) instance goes? In the cloud? On your machine? Into a Docker container?